src/cryptography/hazmat/bindings/openssl/binding.py

```diff
@@ -91,14 +91,20 @@ def _ensure_ffi_initialized(cls):
         with cls._init_lock:
             if not cls._lib_loaded:
                 cls.lib = build_conditional_library(lib, CONDITIONAL_NAMES)
+                # init the OpenSSL locks
+                cls.init_static_locks()
```

Comment:
I'll move init_static_locks back out. Although I think we should do that too it can be in a separate PR.